nengo_loihi/builder.py

```diff
@@ -21,8 +21,14 @@
 from nengo.utils.compat import iteritems
 import nengo.utils.numpy as npext
 
+from nengo_loihi.group import CoreGroup
```

Comment:
To me, `CoreGroup` sounds like a group of cores (which it's not). What about `NeuronGroup`, since it pretty much represents a group of neurons?